### ISSUE TITLE
compare with other pandoc wrappers

### ISSUE THREAD

**Issue by nichtich** [nichtich/Pandoc-Wrapper#12]
Maybe https://github.com/kennethreitz/pyandoc contains some good ideas.

**Comment by bpj** [comment]
It seems to me we are already way ahead of them, with the ability to tie an
object to a specific executable, embedded version and library info etc.

Den 1 nov 2016 20:30 skrev "Jakob Voß" notifications@github.com:

**Comment by nichtich** [comment]
Yes, there are other libraries to compare with. I just created a summary at https://github.com/jgm/pandoc/wiki/Pandoc-wrappers-and-interfaces

**Comment by nichtich** [comment]
See https://github.com/nichtich/Pandoc-Wrapper/issues/13 for a list of other tools to evaluate a look at. A first look did not show any missing features but our documentation could be improved. More features does not mean easier to use!

**Comment by nichtich** [comment]
Looks like none of the other wrappers provide more functionality than this module. The only use case not covered is asynchronous execution but this has to be done with standard Perl fork or event loops anyway. An example may be nice but it's no feature request.
